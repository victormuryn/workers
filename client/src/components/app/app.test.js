// import React from 'react';
// import renderer from 'react-test-renderer';
// import {BrowserRouter as Router} from 'react-router-dom';
// import App from './app';
//
// // jest.mock('react-redux', () => {
// //   const ActualReactRedux = jest.requireActual('react-redux');
// //   return {
// //     ...ActualReactRedux,
// //     useSelector: jest.fn().mockImplementation(() => {
// //       return {user: {}};
// //     }),
// //
// //     useDispatch: jest.fn().mockImplementation(() => {
// //       return () => {};
// //     }),
// //   };
// // });
//
// describe(`App tests`, () => {
//   it(`App not authenticated renders corrects`, () => {
//
//     const tree = renderer.create(<App/>).toJSON();
//
//     expect(tree).toMatchSnapshot();
//   });
//
//   // it(`App authenticated renders corrects`, () => {
//   //   jest.mock('react-redux', () => {
//   //     const ActualReactRedux = jest.requireActual('react-redux');
//   //     return {
//   //       ...ActualReactRedux,
//   //       useSelector: jest.fn().mockImplementation(() => {
//   //         return {
//   //           user: {
//   //             isAuthenticated: true,
//   //           },
//   //         };
//   //       }),
//   //
//   //       useDispatch: jest.fn().mockImplementation(() => {
//   //         return () => {};
//   //       }),
//   //     };
//   //   });
//   //
//   //   const tree = renderer.create(<Router><App /></Router>).toJSON();
//   //
//   //   expect(tree).toMatchSnapshot();
//   // });
// });
