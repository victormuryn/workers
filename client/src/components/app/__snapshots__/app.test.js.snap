// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests App not authenticated renders corrects 1`] = `<div />`;

exports[`App tests App renders corrects 1`] = `<div />`;
